insights/parsers/rhn_conf.py

```diff
@@ -1,20 +1,32 @@
-from .. import Parser, parser, get_active_lines, LegacyItemAccess
+"""
+RHNConf - file ``/etc/rhn/rhn.conf``
+====================================
+
+"""
+
+from insights import Parser, parser, LegacyItemAccess
+from insights.parsers import get_active_lines
 
 
 @parser('rhn.conf')
 class RHNConf(LegacyItemAccess, Parser):
-    """Class to parse the configuration file ``rhn.conf``
+    """
+    Class to parse the configuration file ``rhn.conf``.
+
+    The special feature of ``rhn.conf`` is that values can span multiple
+    lines with each intermediate line ending with a comma.
+
+    This parser uses the :py:class::LegacyItemAccess mix-in to provide access to
```

Comment:
You can use the syntax recommended by @xiangce.  Here's the Sphinx reference: http://www.sphinx-doc.org/en/stable/domains.html#cross-referencing-python-objects